cpp/custrings/text/edit_distance.cu

```diff
@@ -47,12 +47,16 @@ struct editdistance_levenshtein_algorithm
     editdistance_levenshtein_algorithm( custring_view** strings, custring_view** tgts, short* buffer, size_t* offsets, unsigned int* results )
     : d_strings(strings), d_tgt(0), d_tgts(tgts), d_buffer(buffer), d_offsets(offsets), d_results(results) {}
 
+    // pairwise strings
+    editdistance_levenshtein_algorithm( custring_view** strings, short* buffer, size_t* offsets, unsigned int* results )
+    : d_strings(strings), d_tgt(0), d_tgts(0), d_buffer(buffer), d_offsets(offsets), d_results(results) {}
+
     __device__ void operator() (unsigned int idx)
     {
         custring_view* dstr = d_strings[idx];
```

Comment:
Please use clear, unabbreviated names except for widely standard names (like `i` or `it` for a for-loop iterator, `num_` or `n_` as a prefix meaning number). And if you must prefix with `d`, be consistent: `d_string` not `dstr`, `d_target` not `dtgt`. Also don't have two names that only differ in an underscore (`dtgt` vs. `d_tgt`).  `buffer` not `buf`.